content/Rust-1.91.1.md

```diff
@@ -0,0 +1,78 @@
++++
+path = "2025/11/10/Rust-1.91.1"
+title = "Announcing Rust 1.91.1"
+authors = ["The Rust Release Team"]
+aliases = ["releases/1.91.1"]
+
+[extra]
+release = true
++++
+
+The Rust team has published a new point release of Rust, 1.91.1. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.91.1 is as easy as:
+
+```
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][rustup] from the
+appropriate page on our website.
+
+[rustup]: https://www.rust-lang.org/install.html
+
+## What's in 1.91.1
+
+Rust 1.91.1 includes fixes for two regressions introduced in the 1.91.0 release.
+
+### *"Import module mismatch"* linker errors on WASM
```

Comment:
I might recommend rewording this to "Linker and runtime errors on Wasm" to emphasize that this isn't just linker errors but it's also runtime UB in a sense.

Also, a more minor point, but Wasm is abbreviated as "Wasm" as opposed to "WASM" (a very common mistake)